src/tests/ftest/rebuild/container_create_race.yaml

```diff
@@ -42,7 +42,7 @@ ior:
   repetitions: 1
   api: DFS
   dfs_oclass: "RP_2G1"
-  flags: "-w -W -k -G 1"
+  flags: "-w -W -F -k -G 1"
```

Comment:
FYI that you could also increase `np: 8` above for more processes = more files

Reply:
ah, ok